scripts/apps-8-toLowerCaseSubs.js

```diff
@@ -0,0 +1,109 @@
+const dotenv = require('dotenv');  // Application configuration
+const MongoClient = require('mongodb').MongoClient;
+const chalk = require('chalk');  // To color message in console log
+
+/**
+ * Load environment variables from .env file, where API keys and passwords are configured.
+ */
+ dotenv.config({
+ 	path: '.env'
+ });
+
+const processEnv = process.env;
+		
+let dbConn;
+ 
+MongoClient.connect( processEnv.MONGODB_URI || '', {useUnifiedTopology: true} ).then( ( mongoInstance ) => {
+	dbConn = mongoInstance.db( processEnv.MONGODB_NAME || 'subs' );
+	
+	toLowerCaseSubsConfirmed();
+	toLowerCaseSubsExist();
+	toLowerCaseSubsUnconfirmed();
+}).catch( (e) => { console.log( "%s MongoDB ERRROR: %s", chalk.red('✗'), e ) } );
+
+async function toLowerCaseSubsConfirmed() {
+	let numUpperCaseFound = 0;
+	let cursor = dbConn.collection("subsConfirmed").find({});
+
+	while( await cursor.hasNext()){
+		let doc = await cursor.next();
+		console.log("Collection subsConfirmed checking for upperCase against " + doc.email);
+
+		if(doc.email != doc.email.toLowerCase()){
+			console.log("Calling toLowerCase() on: " + doc.email + "\n");
+			numUpperCaseFound++;
+		
+			// toLowerCase() on email containing upperCase
+			const lowerCased = await dbConn.collection("subsConfirmed").findOneAndUpdate(
+				{
+					email: doc.email
+				},
+				{
+					$set: { email: doc.email.toLowerCase() }
+				}
+			)
```

Comment:
Here a propose solution to remove the duplicate, feel free to adapt it.

```suggestion
			).catch( ()=> {
				dbConn.collection("subsConfirmed").findOneAndUpdate(
					{
						email: doc.email
					}
				);
				numDeleted++;
			});
```